opensearch/src/main/java/org/opensearch/sql/opensearch/planner/physical/MLCommonsOperatorActions.java

```diff
@@ -188,4 +195,62 @@ protected MLPredictionOutput getMLPredictionResult(FunctionName functionName,
             .actionGet(30, TimeUnit.SECONDS);
   }
 
+  /**
+   * get ml-commons train, predict and trainandpredict result.
+   * @param inputDataFrame input data frame
+   * @param arguments ml parameters
+   * @param nodeClient node client
+   * @return ml-commons result
+   */
+  protected MLOutput getMLOutput(DataFrame inputDataFrame,
```

Comment:
could also deprecated getMLPredictionResult if it is not required?

Reply:
It is required for now, we can remove it when removing kmeans and ad commands.